### PR TITLE
Add docs for dimap and netcdf conversion blocks

### PR DESCRIPTION
**Pull request by nekhtiari**
Pull Request
============

> Tracked in JIRA by [IN-XXX](https://geoinformationstore.atlassian.net/browse/IN-XXX)
> Paired with @a, @b

### Scope of the PR

Describe the scope of this pull request. Is this a new feature or a bug fix?

### Checklist

- [x] Checked spelling and grammar
- [x] Provided code samples where relevant
- [x] Checked the output of `make html` to ensure new content displays correctly

### CODE REVIEW

File: source/up42-blocks/processing/netcdf-conversion.rst [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+Please see the `block details page <https://marketplace.up42.com/block/87dfdea7-a89a-46b5-8ac3-634ebe26f570>`_ for context.
+
+Block type: ``PROCESSING``
+
+This block converts from the `NetCDF format <https://pro.arcgis.com/en/pro-app/help/data/multidimensional/what-is-netcdf-data.htm>`_ to the ``GeoTIFF`` format. Currently this block can be used with :ref:`Meteomatics <meteomatics-block>` or :ref:`Sentinel-5P Full Scenes <sentinel-5p-block>`.
+
+.. note::
+
+	Bands'structure of the output file depends on the input data structure. In the case of :ref:`Meteomatics <meteomatics-block>`, the bands are realted to the different time steps. For the :ref:`Sentinel-5P Full Scenes <sentinel-5p-block>`, the bands are associated with different variables.
```
Comment (markusuwe) [inline source/up42-blocks/processing/netcdf-conversion.rst:19]: related, not realted

File: source/up42-blocks/processing/netcdf-conversion.rst [hunk 2]
```diff
@@ -0,0 +1,51 @@
+.. meta::
+  :description: UP42 processing blocks: NetCDF conversion block description
+  :keywords: UP42, processing, data format, data type, block description
+
+.. _dimap-conversion-block:
+
+NetCDF Conversion
+===============================
+# TODO UPDATE
```
Comment (markusuwe) [inline source/up42-blocks/processing/netcdf-conversion.rst:9]: Remove todo

File: source/up42-blocks/processing/netcdf-conversion.rst [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+This block converts from the `NetCDF format <https://pro.arcgis.com/en/pro-app/help/data/multidimensional/what-is-netcdf-data.htm>`_ to the ``GeoTIFF`` format. Currently this block can be used with :ref:`Meteomatics <meteomatics-block>` or :ref:`Sentinel-5P Full Scenes <sentinel-5p-block>`.
+
+.. note::
+
+	Bands'structure of the output file depends on the input data structure. In the case of :ref:`Meteomatics <meteomatics-block>`, the bands are realted to the different time steps. For the :ref:`Sentinel-5P Full Scenes <sentinel-5p-block>`, the bands are associated with different variables.
+
+.. note::
+
+	This block can be used mainly to provide a simple visualization for NetCDF data format in other Geospatial tools such as `QGIS <https://qgis.org/en/site/>`_. For any further analysis, we recommend using the NetCDF format itself.
```
Comment (markusuwe) [inline source/up42-blocks/processing/netcdf-conversion.rst:23]: Nice!

File: source/up42-blocks/processing/dimap-conversion.rst [hunk 4]
```diff
@@ -1,16 +1,18 @@
...
+DIMAP Conversion
 ===============================
+# TODO UPDATE
+
 Please see the `block details page <https://marketplace.up42.com/block/87dfdea7-a89a-46b5-8ac3-634ebe26f570>`_ for context.
 
 Block type: ``PROCESSING``
 
-This block converts from the DIMAP format to the ``GeoTIFF`` format.
+This block converts from the `DIMAP format <https://www.intelligence-airbusds.com/en/8722-the-dimap-format>`_ to the ``GeoTIFF`` format. Currently this block can be used with :ref:`SPOT download block <spot-download-block>` or :ref:`Pleiades download block <pleiades-download-block>`.
```
Comment (perusio) [inline source/up42-blocks/processing/dimap-conversion.rst:15]: Can be used with the

File: source/up42-blocks/processing/netcdf-conversion.rst (hunk 1 above)
Comment (perusio) [inline source/up42-blocks/processing/netcdf-conversion.rst:19]: No space after the possessive _Bands'_. Also it reads better:

> The bands' structure of the output file depends on the input data structure.

File: source/up42-blocks/processing/netcdf-conversion.rst [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+Example parameters using the :ref:`Meteomatics <meteomatics-block>` as data source:
```
Comment (perusio) [inline source/up42-blocks/processing/netcdf-conversion.rst:25]: Example job parameters. Please state clearly that this block takes no input parameters. (AFAICT).

**Pull request merged**